packages/core/src/components/breadcrumb-html/breadcrumb.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { ArgsStoryFn } from '@storybook/addons';
```

Comment:
Dit is de "normale" manier om met Storybook te werken.

Maar wij houden niet van normaal ;-)

* Stories leven in `@dso-toolkit/stories`, zie bv. Alert: https://github.com/dso-toolkit/dso-toolkit/blob/master/packages/stories/src/stories/alert.ts
* Implementatie specifieke details worden in de implementatie Storybooks bijgehouden. Alert heeft twee implementaties (CSS en Core): https://github.com/dso-toolkit/dso-toolkit/blob/master/packages/css/stories/alert/alert.stories.ts en https://github.com/dso-toolkit/dso-toolkit/blob/master/packages/core/src/components/alert/alert.stories.ts